test/tck/tck_util.py

```diff
@@ -0,0 +1,120 @@
+import string
+import random
+
+import sys
+
+from neo4j.v1 import GraphDatabase
+
+driver = GraphDatabase.driver("bolt://localhost")
+
+
+def send_string(text):
+    session = driver.session()
+    result = session.run(text)
+    session.close()
+    return result
+
+
+def send_parameters(statement, parameters):
+    session = driver.session()
+    result = session.run(statement, parameters)
+    session.close()
+    return result
+
+
+def get_bolt_value(type, value):
+    if type == 'Integer':
+        return int(value)
+    if type == 'Float':
+        return float(value)
+    if type == 'String':
+        return value
+    if type == 'Null':
+        return None
+    if type == 'Boolean':
+        return bool(value)
+    raise ValueError('No such type : %s', type)
+
+
+def as_cypger_text(expected):
+    if expected is None:
+        return "Null"
+    if isinstance(expected, unicode):
```

Comment:
This won't work across Python 2 and 3. Note how the `string` variable is generated here:
https://github.com/neo4j/neo4j-python-driver/blob/1.0/neo4j/v1/compat.py#L31